multiqc/modules/gatk/analyze_saturation_mutagenesis.py

```diff
@@ -0,0 +1,463 @@
+            "namespace": "GATK",
+        }
+        asm_headers["variants_total_reads"] = {
+            "title": f"Variant reads ({config.read_count_prefix})",
+            "description": f"Reads with a variant called ({config.read_count_desc})",
+            "min": 0,
+            "scale": "Greens",
+            "shared_key": "read_count",
+            "modify": lambda x: x * config.read_count_multiplier,
+            "namespace": "GATK",
+        }
+        asm_headers["filtered_reads"] = {
+            "title": f"Filtered reads ({config.read_count_prefix})",
+            "description": f"Reads filtered from sample ({config.read_count_desc})",
+            "min": 0,
+            "scale": "Reds-rev",
+            "shared_key": "read_count",
+            "modify": lambda x: x * config.read_count_multiplier,
+            "namespace": "GATK",
+        }
+        asm_headers["unmapped_reads"] = {
+            "title": f"Unmapped ({config.read_count_prefix})",
+            "description": f"Unmapped reads in sample ({config.read_count_desc})",
+            "min": 0,
+            "scale": "Purples-rev",
+            "shared_key": "read_count",
+            "hidden": True,
+            "modify": lambda x: x * config.read_count_multiplier,
+            "namespace": "GATK",
+        }
+        asm_headers["lowq_reads"] = {
+            "title": f"LowQ ({config.read_count_prefix})",
+            "description": f"Low quality reads in sample ({config.read_count_desc})",
+            "min": 0,
+            "scale": "Blues-rev",
+            "shared_key": "read_count",
+            "hidden": True,
+            "modify": lambda x: x * config.read_count_multiplier,
+            "namespace": "GATK",
+        }
+        asm_headers["evaluable_reads"] = {
+            "title": f"Evaluable ({config.read_count_prefix})",
+            "description": f"Evaluable reads in sample ({config.read_count_desc})",
+            "min": 0,
+            "scale": "Greys",
+            "shared_key": "read_count",
+            "hidden": True,
+            "modify": lambda x: x * config.read_count_multiplier,
+            "namespace": "GATK",
+        }
+        asm_headers["disjoint_pairs"] = {
+            "title": f"Disjoint ({config.read_count_prefix})",
+            "description": f"Reads from disjoint (non-overlapping) paired-end reads in sample ({config.read_count_desc})",
+            "min": 0,
+            "scale": "Oranges",
+            "shared_key": "read_count",
+            "hidden": True,
+            "modify": lambda x: x * config.read_count_multiplier,
+            "namespace": "GATK",
+        }
+        asm_headers["wt_reads_disjoint"] = {
+            "title": "WT (disjoint) (M)",
+            "description": "WT reads called from disjoint pairs (millions)",
+            "min": 0,
+            "scale": "Purples",
+            "shared_key": "disjoint_read_count",
+            "hidden": True,
+            "modify": lambda x: x / 1000000,
```

Comment:
It would be nice to make all of these user-configurable as well, using a module-specific config.

Reply:
I experimented with it a bit and realised it makes it difficult to read to the table when different prefixes are used for different read counts. E.g. you compare 42 to 42, but in one column it's 42 reads, and in the other one it's 42M reads. So I'd stick to `read_count_*` everywhere - it's already configurable.

Reply:
Sounds good 👍🏻 

Nice compromise actually, as the different `shared_key` strings are there, so the background bar scales will still be grouped, even if the units are shared.